daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -627,7 +630,22 @@ object Main extends Logging {
         val extVarBindingNodeOpt = (configNode \ "externalVariableBindings").headOption
         extVarBindingNodeOpt match {
           case None => Seq.empty
-          case Some(extVarBindingsNode) => ExternalVariablesLoader.getVariables(extVarBindingsNode, tunables)
+          case Some(extVarBindingsNode) => {
+            try {
+              ExternalVariablesLoader.getVariables(extVarBindingsNode, tunables)
+            } catch {
+              case qupe: QNameUndefinedPrefixException => {
```

Comment:
Shoudl be a base class or base trait here. This code doesn't really need to know what kinds of exceptions can come out of the variable bindings, only that one did. 

Your message in the second warning should be generic "Exception when processing external variable binding file".